sickbeard/helpers.py

```diff
@@ -114,65 +114,81 @@ def remove_non_release_groups(name, clean_proper=False):
     # Check your database for funky release_names and add them here, to improve failed handling, archiving, and history.
     # select release_name from tv_episodes WHERE LENGTH(release_name);
     # [eSc], [SSG], [GWC] are valid release groups for non-anime
+
+    # All lowercase and alphabetically sorted
+    groups = {
+        '-20-40',
+        '1044',
+        'agusiq',
+        'androidtwou',
```

Comment:
@ratoaq2 You are absolutely correct, this single regex would not catch 100% of the cases, however a handful of similar regex's (or possibly extending this one, although I'd lean towards a couple of separate regex's instead of a single "mega" regex) should be fairly reliable.  Also the advantage here is it could just as easily provide release group detection, not just non-release group parsing.  

If the number of gotchas for the regex's is relatively few, it might be easier to do:
```
for gotcha in problem_non_groups:
     title = title.replace(gotcha, '')
```
Although I'm not as fond of that approach because of possible unintended consequences when adding new non-groups.